airbyte-notification/src/main/java/io/airbyte/notification/SlackNotificationClient.java

```diff
@@ -33,11 +34,9 @@ public class SlackNotificationClient extends NotificationClient {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SlackNotificationClient.class);
 
-  private final HttpClient httpClient = HttpClient.newBuilder()
-      .version(HttpClient.Version.HTTP_2)
-      .build();
   private final SlackNotificationConfiguration config;
 
+  @JsonCreator
```

Comment:
Why do we need that?

Reply:
This was throwing an error when I tried to construct the SlackNotificationClient without this:

```      
Notification notification = new Notification().withNotificationType(NotificationType.SLACK).withSendOnFailure(false).withSendOnSuccess(false)
          .withSlackConfiguration(slackConfig.get());
      SlackNotificationClient notificationClient = new SlackNotificationClient(notification);
      return notifySchemaChangeActivity.notifySchemaChange(notificationClient, connectionId, isBreaking);
```